infra/base-images/base-runner/bad_build_check

```diff
@@ -58,7 +58,7 @@ function check_instrumentation {
     CHECK_FAILED=$(egrep "ERROR: no interesting inputs were found. Is the code instrumented" -c $FUZZER_OUTPUT)
     if (( $CHECK_FAILED > 0 )); then
       echo "BAD BUILD: $FUZZER does not seem to have coverage instrumentation."
-
+      cat $FUZZER_OUTPUT
```

Comment:
If the output is long, the original BAD BUILD message can get lost in long scroll. Can reverse order, cat first and then echo BAD BUILD. Same for one below.

Reply:
It's a fair point but I think it's weird to print the output without any context and then print the context.